compose/README.md

```diff
@@ -81,6 +89,19 @@ Make commands above, and any subsequent calls to it below can be reviewed using
 from the compose directory:
 
     $ make help
+    
+## Upgrading to the latest version of Archivematica
+
+The installation instructions above will install the submodules defined in https://github.com/artefactual-labs/am/tree/master/src which are from the qa/1.x branches of Archivematica and the Storage Service
```

Comment:
Storage Service's branch name is `qa/0.x` instead.

Period at the end of the sentence is missing.